src/pudl/output/pudltabl.py

```diff
@@ -129,21 +129,39 @@ def __init__(
         # Used to persist the output tables. Returns None if they don't exist.
         self._dfs = defaultdict(lambda: None)
 
-    def pu_eia860(self, update=False):
-        """Pull a dataframe of EIA plant-utility associations.
+    def filter_query(self, table: str):
```

Comment:
Are we anticipating other kinds of selection happening in here? If not I think a more specific name than `filter_query()` would improve readability. Maybe `select_between_dates()`?